plugin/pxr/maya/lib/usdMaya/testenv/testUsdImportXforms.py

```diff
@@ -122,8 +125,7 @@ def testImportMayaXformVariations(self):
             attrVals = {}
             allNodes.append(node)
             allExpected[node] = attrVals
-            for enabled, (attr, (valMin, valMax)) in itertools.izip(enabledArray,
-                                                                    ATTRS.iteritems()):
```

Comment:
Does anything else still use `itertools`, or can we also remove that import from the top of this function?

Reply:
Yes, itertools is still used. See line 121:

` for i, enabledArray in enumerate(itertools.product((False, True), repeat=len(ATTRS))):`

Reply:
Ah, duh. Sorry!